src/components/GoodList.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { GoodCard } from './GoodCard';
+
+type Props = {
+  goods: string[];
+};
+
+export const GoodList: React.FC<Props> = ({ goods }) => (
+  <ul>
+    {goods.map(good => (
+      <GoodCard good={good} key={good} />
```

Comment:
According to the checklist, when rendering a list, you must add a `key` prop that has a unique value. However, using the `good` string as a key is risky if the `goods` array can contain duplicate strings. It's better to use an identifier that is guaranteed to be unique for each element. If `goods` were an array of objects with an `id` property, you would use `good.id` as the key.